src/AmbientSounds.Uwp/Strings/fr-FR/Resources.resw

```diff
@@ -482,7 +485,7 @@
     <value>Bienvenue à la minuterie de mise au point</value>
```

Comment:
Should I translate all of these to what I'd expect, so "minuterie de concentration" or something like that (which is for a person focusing, not a camera focusing?)

Reply:
yes please!